src/test/scala/Constants.scala

```diff
@@ -27,13 +25,18 @@ trait Constants {
       println(v)
     }
   }
-  protected val otherUser = "shunjikonishi" 
+
+  protected val user = sys.env("GITHUB_USER")
+  protected val userRepo = sys.env("GITHUB_REPO")
```

Comment:
Not sure if userRepo is necessary. The repo we used to test the API won't change often if at all. It is already defined in line 17.